src/binder/expression_binder.cpp

```diff
@@ -126,6 +126,15 @@ std::shared_ptr<Expression> ExpressionBinder::implicitCast(
     }
 }
 
+// cast without implicit checking.
+std::shared_ptr<Expression> ExpressionBinder::forceCast(
```

Comment:
well this is exactly implicit cast. User didn't write a `CAST` function but we are casting for them. So `forceCast` is not a good name here. Why are we not reusing implicit cast instead?

Reply:
Implicit cast checks for whether or not the cast follows implicit casting rules, while forceCast does not. Here, forceCast is used for casts that are outside of the rules of implicit casting.